### PR TITLE
updating core with latest contract changes

### PR DESCRIPTION
**Pull request by nickreynolds**
running migrations with latest contract changes, updating wrappers to support changes

### CODE REVIEW

File: packages/core/src/contracts/tcr/parameterizer.ts [hunk 1]
```diff
@@ -274,18 +248,11 @@ export class Parameterizer extends BaseWrapper<ParameterizerContract> {
...
-        return {
-          propID: e.args.propID,
-          paramName: e.args.name,
-          proposedValue: e.args.value,
-          pollID: e.args.pollID,
-        };
-      });
+      ._NewChallengeStream({}, { fromBlock })
+      .concatFilter(async e => this.isChallengeResolved(e.args.challengeID))
+      .map(e => e.args.propID);
```
Comment (walfly) [inline packages/core/src/contracts/tcr/parameterizer.ts]: whats the thought around changing to just the string?

File: packages/core/src/contracts/tcr/parameterizer.ts [hunk 2]
```diff
@@ -525,7 +492,7 @@ export class Parameterizer extends BaseWrapper<ParameterizerContract> {
    * @param propID ID of prop to check
    */
   public async getPropProcessBy(propID: string): Promise<Date> {
-    const expiryTimestamp = await this.instance.getPropProcessBy.callAsync(propID);
+    const [, , , , , expiryTimestamp] = await this.instance.proposals.callAsync(propID);
```
Comment (walfly) [inline packages/core/src/contracts/tcr/parameterizer.ts]: gross

File: packages/core/src/contracts/tcr/parameterizer.ts [hunk 3]
```diff
@@ -534,22 +501,25 @@ export class Parameterizer extends BaseWrapper<ParameterizerContract> {
...
+    return name;
   }
 
   /**
    * Returns the value proposed associated with the given proposal
    * @param propID ID of prop to check
    */
   public async getPropValue(propID: string): Promise<BigNumber> {
-    return this.instance.getPropValue.callAsync(propID);
+    const [, , , , , , value] = await this.instance.proposals.callAsync(propID);
```
Comment (walfly) [inline packages/core/src/contracts/tcr/parameterizer.ts]: dear lord

File: packages/core/src/contracts/tcr/parameterizer.ts (hunk 1 above)
Comment (nickreynolds) [reply packages/core/src/contracts/tcr/parameterizer.ts]: when we updated the event to match AdChain standard, we lost a couple of the parameters in the event that help us make the ParamProp, so we just return the ID and get the ParamProp when you need it.

File: packages/core/src/contracts/tcr/parameterizer.ts (hunk 2 above)
Comment (nickreynolds) [reply packages/core/src/contracts/tcr/parameterizer.ts]: if you have a better way of doing this, let me know. This is generally how we've been doing it though.

**Pull request merged**